package.json

```
@@ -34,14 +34,14 @@
     "http-server": "^0.10.0",
     "jasmine-core": "^2.3.4",
     "jshint": "^2.8.0",
-    "karma": "^1.3.0",
+    "karma": "^1.7.1",
     "karma-chrome-launcher": "^2.0.0",
     "karma-jasmine": "^1.0.2",
     "mkdirp": "^0.5.1",
     "precommit-hook": "^3.0.0",
     "uglify-js": "^3.4.6"
   },
   "dependencies": {
-    "video.js": "^5.6.0 || ^6.2.8 || ^7.0.2"
+    "video.js": "5.x || 6.x || 7.x"
```

This needs `|| 8.x` to properly support v8.
```suggestion
    "video.js": "5.x || 6.x || 7.x || 8.x"
```